packages/ui/__tests__/Radio.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { mountAndCheckA11Y } from '@hazelcast/test-helpers'
+
+import { v4 as uuid } from 'uuid'
+import { Radio } from '../src/Radio'
+
+jest.mock('uuid')
+
+const uuidMock = uuid as jest.Mock<ReturnType<typeof uuid>>
+
+describe('Radio', () => {
+  beforeEach(() => {
+    uuidMock.mockImplementation(() => 'uuidtest')
+  })
```

Comment:
Nit: We can define the `mockImplementation` under the mock. We should not need the `beforeEach`.